lib/webpack.config.js

```diff
@@ -5,20 +5,16 @@ var CopyWebpackPlugin = require('copy-webpack-plugin');
 
 var basePath = __dirname;
 
-var env = process.env.NODE_ENV;
-var production = 'production';
-
 var config = {
   context: path.join(basePath, "src"),
+
   resolve: {
-    // .js is required for react imports.
-    // .tsx is for our app entry point.
-    // .ts is optional, in case you will be importing any regular ts files.
-    extensions: ['', '.js', '.ts', '.tsx']
+    extensions: ['.js', '.ts']
   },
 
   entry: [
-    './index.ts'
+    'es6-promise',
```

Comment:
Since this ibrary will be transpiled as UDMD I don't think the advantage of this.